cpu/stm32_common/periph/rtc.c

```diff
@@ -322,4 +329,257 @@ void ISR_NAME(void)
     cortexm_isr_end();
 }
 
+#elif defined(CPU_FAM_STM32F1)
+
+void rtc_init(void)
+{
+    DEBUG("[RTC rtc_init]\n");
+
+    // Enable APB1 clocks
+    RCC->APB1ENR |= RCC_APB1ENR_PWREN | RCC_APB1ENR_BKPEN;
+
+    // Disable backup domain write protection
+    PWR->CR |= PWR_CR_DBP;
+
+    if ((RCC->BDCR & RCC_BDCR_RTCEN) != RCC_BDCR_RTCEN) // if RTC clock disabled
+    {
+        DEBUG("[RTC initialize]\n");
+
+        // Resets the entire Backup domain
+        RCC->BDCR |= RCC_BDCR_BDRST;
+        // Reset not activated
+        RCC->BDCR &= ~RCC_BDCR_BDRST;
+
+        //RTC clock enabled
+        //LSE oscillator clock used as RTC clock10: LSI oscillator clock used as RTC clock
+        RCC->BDCR |= RCC_BDCR_RTCEN | RCC_BDCR_RTCSEL_LSE;
+
+        // Second interrupt is enabled.
+        RTC->CRH |= RTC_CRH_SECIE;
+
+        // Enter configuration mode.
+        RTC->CRL |= RTC_CRL_CNF;
+
+        //If the input clock frequency (fRTCCLK) is 32.768 kHz, write 7FFFh in this register to get a signal period of 1 second.
```

Comment:
How does this comment relate to your code? You're writing `0x8000` into `RTC->PRLL`, should that be `0x7fff` instead?

Reply:
Yes, 0x7fff will be correct